include/icp/icp.h

```diff
@@ -128,6 +160,16 @@ namespace icp {
         /** The current transform. */
         const RBTransform& current_transform() const;
 
+        /** The matches. */
+        const std::vector<Match>& get_matches() const;
```

Comment:
Do we need this function in this PR? If we do not need it for anything here we don't need to add it for now.